File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
//comments 
int main(void)
{
    //set a float dollars
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
        
    }
    while (dollars < 0);
    if (dollars >= 0)
    {
        //comments
        int cents = round(dollars * 100);
        int a = cents / 25;
        cents -= a * 25;
        int b = cents / 10; 
        cents -= b * 10;
        int c = cents / 5;
        cents -= c * 5;
        int d = cents / 1;
        int e = a + b + c + d; 
        //comments 
        printf("%i\n", e);
        
    }
}

File: pset6/cash/cash.py
from cs50 import get_float

def main():
    cents = get_change()
    print(coins(cents))

    
def get_change():
    while True:
        n = get_float("Changed owed: ")
        if n > 0:
            break
    return round(n * 100)

def coins(cents):
    a = cents // 25
    cents -= a * 25
    b = cents // 10
    cents -= b * 10
    c = cents // 5
    cents -= c * 5
    d = cents 
    return a + b + c + d
    
main()